mantidproject/mantid
# Fixes to codebase for MSVC 2017

Modifications to the codebase to facilitate building with MSVC 2017

**To test:**

- build with MSVC 2017 and ensure everything passes
- run all unit tests
- test output package created with CPACK

Fixes #20598. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
